### ISSUE TITLE
Scoring System

### ISSUE THREAD

**Issue by dkonno** [w3c/silver#340]
I think it's a good idea to calculate the score with the points system to show the suitability of the site.
In WCAG2, for example, missing one of 100 alt texts was a nonconformity. Adopting a points system gives publishers a numerical goal as to how much more effort will be required to make their site more accessible. It also makes it easier for site users to assess how accessible the site is. I believe that it will be beneficial to both the publisher and the site user.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
